src/common/constants/index.ts

```diff
@@ -0,0 +1,49 @@
+export const CATAGORY: string[] = ["한식", "중식", "일식", "양식", "분식", "카페", "편의점", "기타"];
+
+export const ADRESS: string[] = [
```

Comment:
오타 있네요!
추가로 string[]으로 타입을 잡아주기보다 as const로 타입을 좁혀주면 좋을 것 같습니다~
```suggestion
export const CATEGORY: string[] = ["한식", "중식", "일식", "양식", "분식", "카페", "편의점", "기타"];

export const ADDRESS: string[] = [
```

Reply:
수정했습니다!